### PR TITLE
Fix wrong option key

### PR DESCRIPTION
**Pull request by kbond**

### CODE REVIEW

File: tests/Routing/RouterTest.php [hunk 1]
```diff
@@ -59,6 +59,22 @@ public function testI18nRoutingPathWithPrefixExceptDefaultStrategy()
     /**
      * @dataProvider i18nRoutingWithPrefixExceptDefaultStrategyProvider
      */
+    public function testI18nRoutingWithPrefixExceptDefaultStrategyWithDefault($locale)
```
Comment (umpirsky) [inline tests/Routing/RouterTest.php]: Unused parameter `$locale`.

File: tests/Routing/RouterTest.php [hunk 2]
```diff
@@ -59,6 +59,22 @@ public function testI18nRoutingPathWithPrefixExceptDefaultStrategy()
     /**
      * @dataProvider i18nRoutingWithPrefixExceptDefaultStrategyProvider
```
Comment (umpirsky) [inline tests/Routing/RouterTest.php]: We should remove `@dataProvide` for this test.

**Review by umpirsky — APPROVED** [review]
Nice catch, thanks! Will merge after test fixes.

**Comment by kbond** [comment]
All set

**Comment by umpirsky** [comment]
Thanks @kbond!

**Pull request merged**